services/core-web/src/components/mine/MineAlert.js

```diff
@@ -31,11 +32,14 @@ const propTypes = {
 };
 
 export class MineAlert extends Component {
-  state = {
-    loaded: false,
-    activeMineAlert: {},
-    pastMineAlerts: [],
-  };
+  constructor() {
```

Comment:
EsLint was complaining about this.  Would have preferred to refactor to functional, but didn't want to sink too much time into this today.